docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -876,6 +882,48 @@ You could change storage policy after table creation with [ALTER TABLE ... MODIF
 
 The number of threads performing background moves of data parts can be changed by [background_move_pool_size](/docs/en/operations/server-configuration-parameters/settings.md/#background_move_pool_size) setting.
 
+### Nested Dynamic Storage
```

Comment:
btw is there a doc for non-nested dynamic disks? seems no, I did not find, then may add it here as well?